clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1122,10 +1108,10 @@ void handleFailure(Throwable throwable) {
 
     @Override
     public ListTopicsResult listTopics(final ListTopicsOptions options) {
-        final KafkaFutureImpl<Map<String, TopicListing>> topicListingFuture = new KafkaFutureImpl<>();
+        final KafkaFutureImpl<Map<String, TopicListItem>> topicListingFuture = new KafkaFutureImpl<>();
```

Comment:
`TopicListing` seems fine to me if there's no option which is clearly superior. Although the word "listing" may be ambiguous in isolation, there seems little potential for confusion here.